### DISCUSSION TITLE
Missing content-disposition header on Response

### DISCUSSION THREAD

**Discussion by dtiziani** [sindresorhus/got#1945] [Q&A] [answered]
#### Describe the bug

 - Node.js version: 16.13.0
 - OS & version: Ubuntu 21.04

No Content-Disposition Header from a GET request.

#### Actual behavior
Using the browser or node-fetch to any URL, I'm able to retrieve the header "Content-Disposition" correctly.
When using both `got` and `axios`, this header is not present at all, and then I cannot retrieve the filename.

All the resources I've got from Google mentions CORS issues and adjustments on the server that is serving the file, but as I'm running a backend service, I need to download files from servers I have no control whatsoever.

#### Expected behavior
Retrieve the 'content-disposition' from response headers.
...

#### Code to reproduce

```js
const response = await got(
'https://www.renesas.com/br/en/document/apn/ek-ra2e1-example-project-bundle?language=en',
      { responseType: 'buffer', throwHttpErrors: false }
    );
console.log(response.headers['content-disposition']);
...
```

#### Checklist

- [x] I have read the documentation.
- [x] I have tried my code with the latest version of Node.js and Got.

**Comment by szmarczak** [comment, marked as answer]
Not a Got issue. The status code is `403` so this is expected - probably you're missing some headers.
